cpp/dolfinx/mesh/utils.h

```diff
@@ -158,6 +158,22 @@ compute_vertex_coords_boundary(const mesh::Mesh<T>& mesh, int dim,
 
 } // namespace impl
 
+/// @brief Compute the indices of all exterior facets that are owned by
+/// the caller.
+///
+/// An exterior facet (co-dimension 1) is one that is connected globally
+/// to only one cell of co-dimension 0).
+///
+/// @note Collective.
+///
+/// @param[in] topology Mesh topology.
+/// @param[in] facet_type_idx The index of the facet type in
```

Comment:
Passing around indices is a bit confusing. Could we pass the cell type enum?

Reply:
Yes, that's fine, but we do the same thing in a lot of other mixed-topology functions not touched by this PR. Might be better to do them all in a separate PR.

Reply:
I've made an issue so we don't forget.